maintainers/maintainer-list.nix

```diff
@@ -519,6 +519,12 @@
     githubId = 38869148;
     name = "Alex Eyre";
   };
+  algram = {
+    email = "aliasgram@gmail.com";
+    github = "Algram";
+    githubId = 5053729;
+    name = "Alias Gram";
+  };
```

Comment:
Please put this in a separate commit, so there are exactly 2 commits in this PR.

The commit message should be:
- `maintainers: add algram` - for the `maintainers/maintainer-list.nix` changes
- `hyperion-ng: init at 2.0.12` - for other changes